cli/ops/workers.rs

```diff
@@ -132,30 +140,31 @@ fn op_create_worker(
 
   let parent_state = state.clone();
 
+  // TODO(bartlomieju): Isn't this wrong?
   let mut module_specifier = ModuleSpecifier::resolve_url_or_path(specifier)?;
-
-  let mut child_argv = parent_state.argv.clone();
-
   if !has_source_code {
-    if let Some(module) = state.main_module() {
+    if let Some(module) = global_state.main_module() {
```

Comment:
Workers are supposed to resolve entry script specifier relative to parent worker entry script (see `cli/tests/039_worker_deno_ns.ts`). That suggest that `main_module` should be stored per worker and not globally.